sklearn/decomposition/dict_learning.py

```diff
@@ -929,7 +929,9 @@ def fit(self, X, y=None):
         ----------
         X : array-like, shape (n_samples, n_features)
             Training vector, where n_samples in the number of samples
-            and n_features is the number of features.
+            and n_features is the number of features. Ignored.
```

Comment:
No, I checked only `decomposition` and `manifolds`. Maybe I can do `cluster` tonight. Anything else?